ui/dashboards/src/context/DashboardProvider/panel-group-slice.ts

```diff
@@ -57,17 +63,24 @@ export type PanelGroupId = number;
 
 export interface PanelGroupDefinition {
   id: PanelGroupId;
-  items: GridItemDefinition[];
   isCollapsed: boolean;
   title?: string;
+  itemLayouts: PanelGroupLayout[];
+  itemPanelKeys: Record<PanelGroupLayoutId, string>;
 }
 
+export interface PanelGroupLayout extends Layout {
+  i: PanelGroupLayoutId;
```

Comment:
can we call this `id`?

Reply:
This comes from `react-grid-layout` (I basically just redefined the type from `string` so it was more obvious what it actually is), so unfortunately has to stay `i` unless we want to have to transform back and forth with their format.